app/serializers/api/admin/subscription_serializer.rb

```diff
@@ -34,8 +34,16 @@ def customer_email
         object.customer&.email
       end
 
-      def customer_name
-        object.customer&.name
+      def customer_first_name
+        object.customer&.first_name
+      end
+
+      def customer_last_name
+        object.customer&.last_name
+      end
```

Comment:
Does the `SubscriptionSerializer` actually use the `customer_first_name` and `customer_last_name` attributes anywhere? I think we can drop them?